documentation/modules/metrics/proc_metrics-deploying-prometheus-operator.adoc

```diff
@@ -10,23 +10,30 @@ To deploy the Prometheus Operator to your Kafka cluster, apply the YAML bundle r
 
 .Procedure
 
-. Download the resource file from the repository and replace the example `namespace` with your own:
+. Download the `bundle.yaml` resources file from the repository.
 +
 On Linux, use:
 +
-[source,shell,subs=+quotes]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/{SupportedPrometheusVersion}/bundle.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
 +
 On MacOS, use:
 +
-[source,shell,subs=+quotes]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/{SupportedPrometheusVersion}/bundle.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
++
+** Replace the example `namespace` with your own.
++
+ifdef::Downloading[]
+** Use the latest `master` release as shown, or choose a release that is compatible with your version of Kubernetes (see the https://github.com/coreos/kube-prometheus#kubernetes-compatibility-matrix[Kubernetes compatibility matrix]). 
+The Prometheus Operator installation works with Kubernetes 1.18+.
```

Comment:
Is this the latest `master` release? Or which version? I expect the matrix lists more versions than just 1.18.

Reply:
I'll double check. This text is already there.

Reply:
Yes, the `master` release works with Kubernetes 1.18.z. That is my understanding from reading the compatibility matrix and the Quickstart. I'll make this clearer.

The matrix only lists Kubernetes versions up to 1.18.